index.js

```diff
@@ -74,4 +75,43 @@ mOcKiNgCaSe.overrideString = () => {
   return mOcKiNgCaSe;
 }
 
+/**
+ * @param {string|string[]} input The user-given input.
+ * @return If the given input is an array of strings.
+ */
+function isArrayOfStrings(input) {
+  if (!Array.isArray(input)) {
+    return false;
+  }
+
+  if (input.some((value, i) => typeof value !== 'string')) {
+    throw TypeError(`Expected array of strings but got type '${typeof value}' at index ${i}`);
+  }
+  return true;
+}
+
+/**
+ * @param {string} input The string to convert.
+ * @return The given string with random casings.
+ */
+function randomCase(input) {
+  return convert(input, () => Math.round(Math.random()));
+}
+
+/**
+ * 
+ * @param {string} input The string to convert.
+ * @param {function(string, number)} shouldLetterBeUpperCase A function given a character and its index,
+ * which returns if the given letter should be set to uppercase. If false, the letter will be lowercase.
+ * @return The converted string.
+ */
+function convert(input, shouldLetterBeUpperCase) {
+  return input.replace(/./g, (str, i) => {
+    if (shouldLetterBeUpperCase(str, i)) {
```

Comment:
Don't you  think that it look cleaner?!
```suggestion
    return shouldLetterBeUpperCase(str, i) ? str.toUpperCase() : str.toLowerCase();
```